app.js

```diff
@@ -111,7 +108,7 @@ app.use(methodOverride());
 
 app.use(cookieParser());
 app.use(sessions({
-	cookieName: 'vs050',
+	cookieName: 'vs070',
```

Comment:
could we generate this automatically rather than hard code? (I'm assuming this should be linked to vizor version)